### PR TITLE
Improve loading experience

### PR DESCRIPTION
**Pull request by mvaivre**
Wait for the new web3 version to be out and #193 to be merged.

### CODE REVIEW

File: src/api/addressApi.ts [hunk 1]
```diff
@@ -18,14 +18,43 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
...
   addressHash: AddressHash
-): Promise<AddressDataResult> => {
+): Promise<AddressBalanceResult> => {
   const balances = await client.addresses.getAddressesAddressBalance(addressHash)
+
+  return {
+    addressHash,
+    balance: balances.balance,
+    lockedBalance: balances.lockedBalance
+  }
```
Comment (nop33) [inline src/api/addressApi.ts:39]: very nit:
```suggestion
  return {
    addressHash,
    ...balances
  }
```

On second thought, for something as simple as an API call, is creating a method in the api directory really needed? :/ In this particular case, we are simply wrapping the API call.

Comment (nop33) [reply src/api/addressApi.ts:39]: Same for below.

File: src/components/Cards/Card.tsx [hunk 2]
```diff
@@ -28,17 +28,14 @@ export interface CardProps extends HTMLMotionProps<'div'> {
...
-  ) : (
-    <Container className={className} initial={false} {...props}>
-      <header>
-        <LabelText>{label}</LabelText>
-      </header>
-      <Content>{children}</Content>
-    </Container>
-  )
+const Card = ({ label, className, children, isLoading, ...props }: CardProps) => (
+  <Container className={className} initial={false} {...props}>
```
Comment (nop33) [inline src/components/Cards/Card.tsx:32]: nit: the `className` could be removed so that it becomes part of `props`

**Review by nop33 — APPROVED** [review]
Some very nit comments, functional tests pass, approving anyways!

File: src/api/addressApi.ts (hunk 1 above)
Comment (mvaivre) [reply src/api/addressApi.ts]: Done

**Pull request merged**